src/app.py

```diff
@@ -1953,6 +1955,73 @@ def get_dataset_revision_number(id):
     # Response with the integer
     return jsonify(revision_number)
 
+@app.route('/datasets/<id>/revisions', methods=['GET'])
```

Comment:
To be consistent, please add a comment block that describes this call, the parameters, and return type above this line. Follow examples from other endpoints in this file.